src/components/OfferCard/OfferCard.tsx

```diff
@@ -56,6 +60,20 @@ export const OfferCard: FC<TTokensCard> = ({ offer, testid }) => {
     tokenId: offer?.tokenId.toString()
   }), [currentChainId, offer]);
 
+  const tokensCount = useMemo(() => {
```

Comment:
1. move to utils
2. check existing count calculation on tokenPage (no need to duplicate this code)

Reply:
fixed